app.ts

```diff
@@ -0,0 +1,187 @@
+"use strict";
+window.onload = () => { onloadValues() };
+window.onresize = () => { fromAndToValuesAdjustment() };
+
+const rowHeight = 49 + 15;
```

Comment:
You've made this a const (which in Typescript disallows your from editing the value later), but you are adding two numbers?

Also - I am assuming from the previous messages that you have access to the style guide. Pls advise me, does it say something about declaring all of your variables at the top of the file/scope?

Reply:
"Typescript disallows your from editing the value late" not really